CMakeLists.txt

```diff
@@ -245,68 +230,63 @@ add_library(
 
 #-------------------------------------------------------------------------------
 # CUDA support.
+message( "" )
 set( blaspp_use_cuda false )  # output in blasppConfig.cmake.in
 if (gpu_backend MATCHES "^(auto|cuda)$")
-    include( CheckLanguage )
-    check_language( CUDA )
-    if (gpu_backend STREQUAL "cuda" AND NOT CMAKE_CUDA_COMPILER)
-        message( FATAL_ERROR "gpu_backend = ${gpu_backend}, but CUDA was not found")
+    message( STATUS "${bold}Looking for CUDA${not_bold} (gpu_backend = ${gpu_backend})" )
+    if (gpu_backend STREQUAL "cuda")
+        find_package( CUDAToolkit REQUIRED )
+    else()
+        find_package( CUDAToolkit QUIET )
+    endif()
```

Comment:
@ax3l Also, this is what you meant, right?

Reply:
I believe this was resolved, I see a reply from Ax3l, not here but in the main PR page for this